src/test/java/com/thindeck/dynamo/DyTasksTest.java

```diff
@@ -68,8 +76,106 @@ public void getTask() throws Exception {
     }
 
     /**
+     * DyTask can add a task with attributes in NULL.
+     * @throws Exception In case of error.
+     */
+    @Test
+    @Ignore
```

Comment:
Because this test fail with the add method. This method was already implemented before that i take this task and the bug was reported https://github.com/yegor256/thindeck/issues/491